kuma/static/styles/includes/_vars.scss

```diff
@@ -177,18 +177,18 @@ break points
 - divide pixels by browser default font size (16px)
 - and multiply by 1em to convert unit to em.
 */
-$max-width-default: (1399px / 16px) * 1em;
-$small-desktop-ends: (1199px / 16px) * 1em;
-$tablet-ends: (1023px / 16px) * 1em;
-$mobile-ends: (767px / 16px) * 1em;
-$small-mobile-ends: (479px / 16px) * 1em;
+$max-width-default: (1399px / 15px) * 1em;
+$small-desktop-ends: (1199px / 15px) * 1em;
+$tablet-ends: (1023px / 15px) * 1em;
+$mobile-ends: (767px / 15px) * 1em;
+$small-mobile-ends: (479px / 15px) * 1em;
```

Comment:
Nit: Is it possible to move ``15px`` into a SCSS variable? This change from ``16px`` to ``15px`` would have been a one-line change...

Reply:
Agreed. It is dividing by the base body font size. I would suggest updating the variable here https://github.com/mozilla/kuma/blob/master/kuma/static/styles/includes/_vars.scss#L132 to be `15px` and then use the variable for consistency.

Reply:
All set!